models/branches.go

```diff
@@ -581,7 +580,7 @@ func RemoveDeletedBranch(repoID int64, branch string) error {
 }
 
 // RemoveOldDeletedBranches removes old deleted branches
-func RemoveOldDeletedBranches(ctx context.Context, olderThan time.Duration) {
+func RemoveOldDeletedBranches(olderThan time.Duration) {
```

Comment:
Please don't remove the context here. 

We need to keep this because we're going to start pushing context through the stack in order to simplify things like process hierarchies, logging and db transactions.